### PR TITLE
add tizen5.0 for what is tizen

### PR DESCRIPTION
**Pull request by jhchoilee**
I want add tizen5.0 for what is tizen
After reviewing this document, and then, I will add this page to ToC

### CODE REVIEW

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Tizen 5.0
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: Kindly add file to ToC.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  .Corefx 2.1.1 is upgraded to 2.1.4.
+  .Xamarin.Froms is upgraded to 3.2.0.
+Etc.
+
+* **Tizen Circular UI for Watch** : Tizne 5.0 provides easy watch app development with Tizen Circular UI APIs and provides "Tizne Wearalbe App" project template for creating Watch circular applicaiton easily.
+
+![img](media/5.0_introduction_TizenCircularUIforWatch.png)
+![img](media/5.0_introduction_TizenCircularUIforWatch_template.png)
+
+* **C# API enhancement in TizenFX**: Many APIs in TizenFX are added.
```
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: Also, kindly verify the alignment of the images.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Tizen 5.0 M2
+
+
+[The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was released in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently Tizen is used for the various kinds of smart devices.
+Adding the enhancement on service with Tizen 5.0 provides the opportunity to develop applications which are related to service centric. When it comes to users, they will get various experiences with more enhanced applications. Tizen 5.0 provides APIs such as TIDL(Tizen Interface Definition Language), RPC-port for easy integration of various kinds of services (eg. Smart-things ) and core open sources such as Blink, GStreamer, EFL, Wayland etc. are upgraded. So the latest standard specification can be used in Tizen.
+
+The key features are as follows:
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Tizen 5.0 M2
+
+
+[The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was released in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently Tizen is used for the various kinds of smart devices.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+   - Nlp API set for new Namespace and Class.
+   - InputMethod API set for floating keyboard and to hide or launch IME.
+   - Wi-Fi API set for forgetting access point (AP), BssidScan, RawSsid, CountryCode, and WPS connection cancellation.
+   - NUI API set for KeyboardRepeatInfo and TextPredition...
+
+    and so on.
+
+  ![img](media/5.0_introduction_TizenFXAPI_v5.png)
+
+For more information about open source upgrade, see release note [M1](../../release-notes/tizen-5-0-m1.md) and [M2](../../release-notes/tizen-5-0-m2.md).
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  .Xamarin.Froms is upgraded to 3.2.0.
+Etc.
+
+* **Tizen Circular UI for Watch** : Tizne 5.0 provides easy watch app development with Tizen Circular UI APIs and provides "Tizne Wearalbe App" project template for creating Watch circular applicaiton easily.
+
+![img](media/5.0_introduction_TizenCircularUIforWatch.png)
+![img](media/5.0_introduction_TizenCircularUIforWatch_template.png)
+
+* **C# API enhancement in TizenFX**: Many APIs in TizenFX are added.
+   - MediaController API set for playlist, event, and capabilities...
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+GStreamer, Pulseaudio are upgraded to latest stable release
+    EFL version 1.16 is upgraded to 1.20
+Wayland is upgraded to 1.15.0 version.
+.NET Core(Runtime) is upgraded
+  .Coreclr 2.1.1 is upgraded to 2.1.4.
+  .Corefx 2.1.1 is upgraded to 2.1.4.
+  .Xamarin.Froms is upgraded to 3.2.0.
+Etc.
+
+* **Tizen Circular UI for Watch** : Tizne 5.0 provides easy watch app development with Tizen Circular UI APIs and provides "Tizne Wearalbe App" project template for creating Watch circular applicaiton easily.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+# Tizen 5.0 M2
+
+
+[The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was released in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently Tizen is used for the various kinds of smart devices.
+Adding the enhancement on service with Tizen 5.0 provides the opportunity to develop applications which are related to service centric. When it comes to users, they will get various experiences with more enhanced applications. Tizen 5.0 provides APIs such as TIDL(Tizen Interface Definition Language), RPC-port for easy integration of various kinds of services (eg. Smart-things ) and core open sources such as Blink, GStreamer, EFL, Wayland etc. are upgraded. So the latest standard specification can be used in Tizen.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+The key features are as follows:
+* **TIDL, rpc-port**: TIDL is Tizen Interface Definition Language. Service application can communicate with other service application or service deamon with this API set. Rpc-port provides application IPC mechanism.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+Applications can expose RPC style service interface to other applications with TIDL.
+
+* **Core Open source Upgrade**: In Tizen 5.0, many core open sources are upgraded as follows:
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md (hunk 1 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: I will raise the new PR after finishing "add tizen5.0 for what is tizen" and "add tizen5.5 for what is tizen"

**Comment by jhchoilee** [comment]
I commit new one based on the review comment.

about "adding ToC for Tizen 5.0 & Tizen 55",
I will raise the new PR after finishing "add tizen5.0 for what is tizen" and "add tizen5.5 for what is tizen"

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+     - Enhancement in web performance such as V8 : ES6 performance enhancement, Web Assembly default enabling.
+   - GStreamer, Pulseaudio are upgraded to latest stable release
+   - EFL 1.16 is upgraded to 1.20
+   - Wayland is upgraded to 1.15.0 version.
+   - .NET Core(Runtime) is upgraded
+    - Coreclr 2.1.1 is upgraded to 2.1.4.
+    - Corefx 2.1.1 is upgraded to 2.1.4.
+    - Xamarin.Froms is upgraded to 3.2.0.
+
+    and so on.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: @jhchoilee  request you to align the list properly.

cap

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+* **Tizen Interface Definition Language (TIDL) and RPC-port**: TIDL enables the service applications to communicate with other service application or service daemon and the RPC-port provides application Inter process Communication (IPC) mechanism.
+Applications can expose RPC style service interface to other applications with TIDL.
+
+* **Core Open source Upgrade**: In Tizen 5.0, the following core open source components are upgraded:
+   - Blink M56 which is the core web engine in Tizen is upgraded to M63.
+     - Enhancement to follow W3C Standards such as Animated PNG, Full featured Indexed DB, CSS grid layout, web USB, MSE, EME, and so on.
+     - Enhancement in standard web features WebRTC, and ARIA 1.1
+     - Enhancement in web performance such as V8 : ES6 performance enhancement, Web Assembly default enabling.
+   - GStreamer, Pulseaudio are upgraded to latest stable release
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: - GStreamer and Pulseaudio are upgraded to the latest stable release

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+   - EFL 1.16 is upgraded to 1.20
+   - Wayland is upgraded to 1.15.0 version.
+   - .NET Core(Runtime) is upgraded
+    - Coreclr 2.1.1 is upgraded to 2.1.4.
+    - Corefx 2.1.1 is upgraded to 2.1.4.
+    - Xamarin.Froms is upgraded to 3.2.0.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: - Xamarin.Forms are upgraded to 3.2.0, and so on.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+  - AudioManager API set for USB Audio output device.
+  - AudioIO API set and enum for sample type and changed maximum sample rate.
+  - MediaPlayer API set to set or get the video ROI area.
+  - Multimedia API set for the rotation and zoom level.
+  - NLP API set for new Namespace and Class.
+  - InputMethod API set for floating keyboard and to hide or launch Input Method Editor (IME).
+  - WiFi API set for forgetting access point (AP), BSSID Scan, RawSSID, CountryCode, and WPS connection cancellation.
+  - NUI API set for KeyboardRepeatInfo, TextPredition and so on.
+
+      and so on.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: Line 41: Remove 
and so on.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md (hunk 11 above)
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: Remove: 
and so on.

Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: I updated with new commit. 
Please check it and let me know the detail what you want.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+    - Enhancement in standard web features WebRTC, and ARIA 1.1
+    - Enhancement in web performance such as V8 : ES6 performance enhancement, Web Assembly default enabling.
+  - GStreamer 1.6.1 is upgraded to 1.12.2.
+  - Pulseaudio 5.0 is upgraded to 11.1.
+  - EFL 1.16 is upgraded to 1.20
+  - Wayland is upgraded to 1.15.0 version.
+  - .NET Core(Runtime) is upgraded
+    - Coreclr 2.1.1 is upgraded to 2.1.4.
+    - Corefx 2.1.1 is upgraded to 2.1.4.
+  - Xamarin.Froms is upgraded to 3.2.0. and so on.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: @jhchoilee 

Slight punctuation additions, spell mistake: 
Copy paste the below sentences at their appropriate places. 

   - Enhancement in standard web features WebRTC and ARIA 1.1.

   - Enhancement in web performance such as V8 : ES6 performance enhancement and Web Assembly default enabling.
- EFL 1.16 is upgraded to 1.20,

- .NET Core(Runtime) is upgraded.


- Xamarin.Forms is upgraded to 3.2.0, and so on.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+
+* **C# API enhancement in TizenFX**: any new APIs are added in TizenFX, and some of the added APIs are as follows:
+  - MediaController API set for playlist, event, and capabilities.
+  - AudioManager API set for USB Audio output device.
+  - AudioIO API set and enum for sample type and changed maximum sample rate.
+  - MediaPlayer API set to set or get the video ROI area.
+  - Multimedia API set for the rotation and zoom level.
+  - NLP API set for new Namespace and Class.
+  - InputMethod API set for floating keyboard and to hide or launch Input Method Editor (IME).
+  - WiFi API set for forgetting access point (AP), BSSID Scan, RawSSID, CountryCode, and WPS connection cancellation.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: - WiFi API set for forget access point (AP), BSSID Scan, RawSSID, CountryCode, and WPS connection cancellation.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+  - NUI API set for KeyboardRepeatInfo, TextPredition and so on.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: - NUI API set for KeyboardRepeatInfo, TextPredition, and so on.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md (hunk 15 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: ok

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
+# Tizen 5.0
+
+[The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was announced in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently, Tizen is used for various kinds of smart devices.
+Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL), RPC-port for easy integration of various kinds of services such as Smart-things. Core open sources such as Blink, GStreamer, EFL, Wayland, and so on are upgraded.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: I changed like this. "Many core open sources are upgraded in Tizen 5.0."

**Comment by jhchoilee** [comment]
I updated with new commit.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 # Tizen 5.0
 
 [The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was announced in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently, Tizen is used for various kinds of smart devices.
-Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL), RPC-port for easy integration of various kinds of services such as Smart-things. Core open sources such as Blink, GStreamer, EFL, Wayland, and so on are upgraded.
+Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL) and RPC-port for easy integration of various kinds of services such as SmartThings. Many core open sources are upgraded in Tizen 5.0.
```
Comment (nawab-samsung) [inline docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: @jhchoilee  this sentence seems incomplete

Can we use components.  
Many core open source components are upgraded in Tizen 5.0.

File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+    - Enhancement in standard web features WebRTC and ARIA 1.1.
+    - Enhancement in web performance such as V8 : ES6 performance enhancement and Web Assembly default enabling.
+  - GStreamer 1.6.1 is upgraded to 1.12.2.
+  - Pulseaudio 5.0 is upgraded to 11.1.
+  - EFL 1.16 is upgraded to 1.20.
+  - Wayland is upgraded to 1.15.0 version.
+  - .NET Core(Runtime) is upgraded.
+    - Coreclr 2.1.1 is upgraded to 2.1.4.
+    - Corefx 2.1.1 is upgraded to 2.1.4.
+  - Xamarin.Forms is upgraded to 3.2.0, and so on.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: "Xamarin.Forms" is the open source project name. 
So I think "is" is right. Xamarin.Forms is proper noun.

[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/platform/what-is-tizen/versions/tizen-5-0-m2.md]: Thanks for clarification.
If you treat this a proper noun, then "is" stands valid. 
Thanks again @jhchoilee

**Comment by jhchoilee** [comment]
Can you merge this pull request? Do I have to do something more???

**Pull request merged**